### ISSUE TITLE
Gtk3: Drag&Drop of pages fails

### ISSUE THREAD

**Issue by genodeftest** [jaap-karssenberg/zim-desktop-wiki#385]
Steps to reproduce:
1. have more than one page in Zim
2. try to drag&drop one page into another one (left sidebar)

What happens:
drag&drop does not happen. Traceback:

```
__init__.py:495:do_drag_data_get:AttributeError: 'SelectionData' object has no attribute 'target'

Traceback (most recent call last):
  File "/path/to/zim/plugins/pageindex/__init__.py", line 495, in do_drag_data_get
    assert selectiondata.target == INTERNAL_PAGELIST_TARGET_NAME
AttributeError: 'SelectionData' object has no attribute 'target'

Local variables in innermost frame:
time: 0
info: 2
selectiondata: <Gtk.SelectionData object at 0x7fcc18324828 (GtkSelectionData at 0x5560f8512b00)>
dragcontext: <__gi__.GdkWaylandDragContext object at 0x7fcc1833de10 (GdkWaylandDragContext at 0x5560f8192830)>
self: <pageindex.PageTreeView object at 0x7fcc183140d8 (zim+plugins+pageindex+PageTreeView at 0x5560f81d83e0)>
```

Version:
Latest git revision from `next` branch as of now.

**Comment by genodeftest** [comment]
I created a fix (valid and correct in terms of Gtk+/GObject documentation), but it still fails due to a bug in GObject python bindings: https://gitlab.gnome.org/GNOME/pygobject/issues/104 and https://gitlab.gnome.org/GNOME/pygobject/issues/90

**Comment by genodeftest** [comment]
https://gitlab.gnome.org/GNOME/pygobject/issues/104 has a workaround which can be found at https://github.com/virtuald/pygi-treeview-dnd. See also: exaile/exaile#200

**Comment by jaap-karssenberg** [comment]
Believe this issue is closed, so issue #390 for blocking issue on dnd

**Comment by swestlake** [comment]
version 0.71.1 has the same regressive bug in that attempting to moving nodes in the index pane does not perform the move of the topic page to another location.

libgtk-3-0:amd64    3.22.11-1
python3-gi              3.22.0-2 

debian linux 64-bit..
